### PR TITLE
Enabling Basic Information Cluster in the ota-provider app

### LINKED ISSUE #14764: ota-provider cannot be commissioned with chip-tool
#### Problem
Commissioning the ota-provider-app fail, when using the CHIP-tool. Without the Basic Information Cluster enabled, the CHIP-tool fails commissioning after it tries to read the VendorID attribute.

 #### Proposed Solution
Enable the Basic Information Cluster using ZAP and commissioning works fine.

### PR DESCRIPTION
**Pull request by harsha-rajendran**
#### Problem
#14764
Commissioning the ota-provider-app fail, when using the CHIP-tool. Without the Basic Information Cluster enabled, the CHIP-tool fails commissioning after it tries to read the VendorID attribute.

#### Change overview
Used ZAP tool to enable the Basic Information Cluster in the ota-provider app.

#### Testing
- Verified commissioning of the ota-provider app is successful with the change.

### CODE REVIEW

File: examples/light-switch-app/light-switch-common/light-switch-app.matter [hunk 1]
```diff
@@ -640,6 +640,13 @@ client cluster Groups = 4 {
   request struct ViewGroupRequest {
     INT16U groupId = 0;
   }
+
```
Comment (hrajendran-samsung) [inline examples/light-switch-app/light-switch-common/light-switch-app.matter:643]: zap-regen updated the light-switch-app, as well.

Comment (tcarmelveilleux) [reply examples/light-switch-app/light-switch-common/light-switch-app.matter:643]: That is a flake I think. Please re-run zap

**Review by tcarmelveilleux — APPROVED** [review]

**Review by carol-apple — APPROVED** [review]

**Review by selissia — APPROVED** [review]

File: examples/light-switch-app/light-switch-common/light-switch-app.matter (hunk 1 above)
Comment (hrajendran-samsung) [reply examples/light-switch-app/light-switch-common/light-switch-app.matter]: Looks like @woody-apple did that! All set.

**Pull request merged**